packages/client/components/MeetingSelector.tsx

```diff
@@ -56,7 +55,6 @@ const MeetingSelector = (props: Props) => {
   useSubscription('MeetingSelector', OrganizationSubscription)
   useSubscription('MeetingSelector', TaskSubscription)
   useSubscription('MeetingSelector', TeamSubscription)
-  useSubscription('MeetingSelector', MeetingSubscription, {meetingId})
```

Comment:
this occurs before the meeting member is created, so the subscription would fail. moved this to useAutoCheckin